testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -436,3 +437,39 @@ def test_pathlib():
     # we really only care that pathlib
     # object handling worked
     assert u.atoms.n_atoms == 3341
+
+
+@pytest.fixture
+def large_dcdfile(tmpdir):
+    # creates a >2Gb DCD file
+    fsize = 3.8  # mb
+    nreps_reqs = int(2100 // fsize)  # times to duplicate traj to hit 2.1Gb
+
+    newf = str(tmpdir / 'jabba.dcd')
+
+    u = mda.Universe(PSF, DCD)
+
+    print(nreps_reqs)
+
+    with mda.Writer(newf, n_atoms=len(u.atoms)) as w:
+        for _ in range(nreps_reqs):
+            for ts in u.trajectory:
+                w.write(u.atoms)
+
+    yield newf, nreps_reqs
+
+
+@pytest.mark.skipif(not os.environ.get('LARGEDCD', False),
```

Comment:
That's kinda confusing logic, and looks undocumented, are we really expecting to use it?

Reply:
I can just remove the test if you like, it was handy while I was fixing the bug

Reply:
test would be good to keep, I just don't really know why you'd need a skipif that isn't really documented. Did we not already have a high memory flag from the EDR tests? Can we just that instead?

Reply:
We should run the test in at least one runner every time. 

And as I said in the original issue, eventually every reader should be tested with a large trajectory so that we have a better chance catching these kind of issues.